.env

```diff
@@ -0,0 +1 @@
+REACT_APP_OPEN_CAGE_API_KEY = 3a1eb076a3534d3b82d064a21f65b260
```

Comment:
why have you added this api key for this feature?

Reply:
deleted